.github/dependabot.yml

```diff
@@ -20,39 +20,39 @@ updates:
 
 # Scanning is disabled for files in /test/ to avoid false positives.
 # These files are used for testing; vulnerable code is never installed or used.
-
-  - package-ecosystem: cargo
-    directory: /test/language_data
-    schedule:
-      interval: monthly
-    ignore:
-      - dependency-name: "*"
-
-  - package-ecosystem: bundler
-    directory: /test/language_data
-    schedule:
-      interval: monthly
-    ignore:
-      - dependency-name: "*"
-
-  - package-ecosystem: gomod
-    directory: /test/language_data
-    schedule:
-      interval: monthly
-    ignore:
-      - dependency-name: "*"
-
-  - package-ecosystem: pip
-    directory: /test/language_data
-    schedule:
-      interval: monthly
-    ignore:
-      - dependency-name: "*"
-
-  - package-ecosystem: maven
-    directory: /test/language_data
-    schedule:
-      interval: monthly
-    ignore:
-      - dependency-name: "*"
-
+# These are commented out because they caused problems with other automated checks
```

Comment:
This might just be my preference, should these lines all just be deleted? They can always be restored from git history, just a little clutter avoidance :)

Reply:
I'm leaving this commented out rather than removing it so I don't forget why this was a bad idea and try it again in 6 months, or have someone else propose it when I gripe about the process. I guess that would be just as well handled by a single comment line, but I don't mind a bit of clutter in this case.